server/container_create_linux.go

```diff
@@ -58,17 +57,15 @@ var (
 
 // cgroupv1HasHugetlb returns whether the hugetlb controller is present on
 // cgroup v1.
-func cgroupv1HasHugetlb() (bool, error) {
+func cgroupv1HasHugetlb() bool {
 	_cgroupv1HasHugetlbOnce.Do(func() {
 		if _, err := ioutil.ReadDir("/sys/fs/cgroup/hugetlb"); err != nil {
-			_cgroupv1HasHugetlbErr = errors.Wrap(err, "readdir /sys/fs/cgroup/hugetlb")
 			_cgroupv1HasHugetlb = false
```

Comment:
IDK, but might it be useful to have a debug log print out here if we set this to false?

Reply:
good idea, added

Reply:
ach, nevermind, the function call flow doesn't make this simple or clear. I think we're okay with out.